firefox-ios/nimbus-features/remoteTabManagement.yaml

```diff
@@ -0,0 +1,22 @@
+# The configuration for the remoteTabManagement feature
+features:
```

Comment:
I believe there is a specific PR review process for new Nimbus features but @adudenamedruby could confirm for certain.

Reply:
Nope, no fancy process for Nimbus; this looks fine to me.
You might be thinking of adding telemetry, where you'd need a data review. :)